seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -1718,6 +1920,20 @@ def test_sv_wgs_get_es_variants(self):
             }}
         ], index=SV_WGS_INDEX_NAME)
 
+    @urllib3_responses.activate
+    def test_mito_get_es_variants(self):
+        self.families = Family.objects.filter(guid='F000014_14')
+        setup_responses()
+        search_model = VariantSearch.objects.create(search={})
+        results_model = VariantSearchResults.objects.create(variant_search=search_model)
+        results_model.families.set(self.families)
+
+        variants, _ = get_es_variants(results_model, num_results=2)
+        self.maxDiff = None
+        self.assertListEqual([v for v in variants if v['chrom'] == 'M'], [PARSED_MITO_VARIANT])
+
+        self.assertExecutedSearch(size=4, index=','.join([SV_WGS_INDEX_NAME, MITO_WGS_INDEX_NAME]))
```

Comment:
this should also be searching in the variants index, not just SV and mito

Reply:
The fixture data for family 14 have only SV and MITO samples.

Reply:
so we should use a different family. Its crucial to test we can search in both